04_TS_decorator/classes/decorator.ts

```diff
@@ -0,0 +1,42 @@
+import { Actions } from "../helpers/helpers";
+import { Student } from "../types/types";
+import { Univercity } from "./building";
+
+interface Commander {
+    execute(action: string, ...args: unknown[]): void;
+}
+
+export class UniversityCommander implements Commander {
+    
+    public univercity: Univercity;
```

Comment:
This property should be either `protected` or `private`. One of the Decorator pattern features is to hide a base interface functionality.